app/src/main/java/org/xbmc/kore/host/HostConnectionObserver.java

```diff
@@ -592,9 +593,29 @@ public void onSuccess(ArrayList<GetPlaylist.GetPlaylistResult> result) {
                 if (result.isEmpty())
                     return;
 
-                for (PlaylistEventsObserver observer : playlistEventsObservers) {
-                    observer.playlistsAvailable(result);
+                boolean isPlaylistChanged = false;
```

Comment:
I think this code could use a comment explaining what it is doing, as it is not immediately obvious.

Reply:
done.